setup.py

```diff
@@ -1,4 +1,9 @@
-from setuptools import setup, Extension
+try:
+    from setuptools import setup, Extension, find_packages
+except:
+    from distutils.core import setup, Extension
+    find_packages = lambda: ['netsnmp']
```

Comment:
Please rebase if you would be so kind, I have removed the use of `find_packages` :smile: